src/controllers/accomodation.controller.js

```diff
@@ -0,0 +1,62 @@
+import response from '../helpers/response.helper';
+import accomodation from '../services/accomodation.services';
+import UserServices from '../services/user.service';
+
+/**
+* Class for Travel Admins dealing with Accomodation
+*/
+class AccomdationController {
+  /**
+   * This function helps to to filter and store accomodation information
+   * @param { String } name name of accomodation
+   * @param { String } description description of accomodation
+   * @param { Integer } locationId location of accomodation
+   * @param { String } category category of accomodation
+   * @param { String } owner owner of accomodation
+   * @param { Array } images images of accomodation
+   * @param { Array } rooms rooms of accomodation
+   * @returns {Object} return created Accomodation
+   */
+  static async accomodationInfo(name, description, locationId, category, owner, images, rooms) {
```

Comment:
Nice job Jaja! however, I would like to suggest that you try to rename this function a name that much with the action. example `saveAccomodationInfo`. What do you think about that?

Reply:
@VedaDom i think that is a good idea